poc/prg.py

```diff
@@ -106,13 +106,24 @@ class PrgSha3(Prg):
     def __init__(self, seed, dst, binder):
         # `dst` is used as the customization string; `seed || binder` is
         # used as the main input string.
-        self.shake = cSHAKE128.new(custom=dst)
+        self.shake = SHAKE128.new()
+        dst_length_l = (len(dst).bit_length() + 7)//8
+        dst_length = len(dst).to_bytes(dst_length_l, byteorder='little')
+        dst_length_l = self.__to_bytes__(dst_length_l)  # one byte value
```

Comment:
Similarly here (`byte` is in the `common` module)

```suggestion
        dst_length_l = byte(1)
```